recipes/hieroglyph/meta.yaml

```diff
@@ -0,0 +1,48 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
```

Comment:
Please clean up comments in the meta.yaml

Reply:
Sorry -- I made a mess of this using the wrong branch.

The Hieroglyph package is in #5498

With a clean meta.yaml.

Reply:
haha ok.  Should we close this then?